docs/guides/depositor-bot.md

```diff
@@ -0,0 +1,74 @@
+# Lido depositor bot
+
+## Introduction
+
+Depositor bot is a part of Deposit Security Module.
+
+The Depositor Bot obtains signed deposit messages from Council Daemons. Once a sufficient number of messages is
```

Comment:
Killer feature of deposit bot is gas price checking add some details how it works.
Also add to envs list gas price envs

Reply:
Added

Reply:
+to @F4ever 
see no mentions of this as of now

Reply:
@TheDZhon please check `How to use` section